src/components/v5/common/ActionSidebar/partials/forms/ManageTokensForm/hooks.ts

```diff
@@ -1,22 +1,30 @@
 import { useCallback, useMemo } from 'react';
 import { Id } from '@colony/colony-js';
+import { useWatch } from 'react-hook-form';
 import { ActionTypes } from '~redux';
 import { mapPayload, pipe } from '~utils/actions';
-import { useAppContext, useColonyContext, useEnabledExtensions } from '~hooks';
+import { useAppContext, useColonyContext } from '~hooks';
 import { ActionFormBaseProps } from '../../../types';
-import { useActionFormBaseHook } from '../../../hooks';
+import {
+  DecisionMethod,
+  DECISION_METHOD,
+  useActionFormBaseHook,
+} from '../../../hooks';
 import { getTokenManagementDialogPayload } from '~common/Dialogs/TokenManagementDialog/helpers';
 import { notNull } from '~utils/arrays';
 import { validationSchema, ManageTokensFormValues } from './consts';
 import { useAdditionalFormOptionsContext } from '~context/AdditionalFormOptionsContext/AdditionalFormOptionsContext';
+import { DECISION_METHOD_FIELD_NAME } from '~v5/common/ActionSidebar/consts';
```

Comment:
All of these should be before imports like `../../../hooks` and after library imports